packages/styles/variables.css

```
@@ -37,10 +37,10 @@
   --accent-secondary-active: var(--gray-30);
   --focus-light: #c11bde;
   --focus-dark: #eb94ff;
-  --issue-critical: #ed5959;
-  --issue-serious: #f3826b;
-  --issue-moderate: #ffdd76;
-  --issue-minor: #d3dde0;
+  --issue-critical: #fe6d6b;
```

This color already exists:

```suggestion
  --issue-critical: var(--accent-danger);
```